src/components/Validator.js

```diff
@@ -47,12 +47,13 @@ export const Validator = {
       }
     });
 
-    const customErrorMessage = _.get(component, 'component.validate.customMessage');
-    if (result && customErrorMessage) {
-      result = component.t(customErrorMessage, {
-        data: component.data
+    const customMessage = _.get(component, 'component.validate.customMessage');
+     const validateCustom = _.get(component, 'component.validate.custom');
+     if (result && (customMessage || validateCustom)) { 
+      result = component.t(result, { 
```

Comment:
This now seems to be ignoring the ```customErrorMessage```. So what I am seeing here is if I have the following within custom....

```valid = input === 'test' ? true : 'You must enter "test"'```

and then have the following set for customErrorMessage...

```
You need to input "test" for this to work.
```

This shows that my customErrorMessage is ignored since it is just going to give me the translation of the result of the validation i.e 'You must enter "test"'.  This breaks all implementations that rely on the customErrorMessage.